srv/components/csm/init.sls

```diff
@@ -19,7 +19,9 @@
 include:
   - components.csm.prepare
   - components.csm.install
-  - components.csm.config
+  - components.csm.config.init_mod
+ # - components.csm.config.post_install
+ # - components.csm.config.config
```

Comment:
If they're included inside `init_mod`, we can delete these lines from common `init`, instead of commenting.